packages/trader/src/App/Components/Elements/ContractDrawer/contract-drawer.jsx

```diff
@@ -56,7 +56,6 @@ const ContractDrawer = observer(
                 is_accumulator={is_accumulator}
                 is_dark_theme={is_dark_theme}
                 is_multiplier={is_multiplier}
-                is_smarttrader_contract={is_smarttrader_contract}
```

Comment:
We don't need this props. From `<ContractAudit />` props are passed to` <ContractDetails />` and there `is_smarttrader_contract` is imported from `@deriv/shared`